### PR TITLE
Create Python-Function-POW.md

### PR DESCRIPTION
**Pull request by nj4710**
For issue #819

### CODE REVIEW

File: Python-Function_POW.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
+# Python 'pow(x,y)'
+
+pow(x, y[, z]) is a built in function in Python 3 to calculate x to the power y and if z is present , returns x to the power y modulo [https://processing.org/reference/modulo.html] z
```
Comment (theflametrooper) [inline Python-Function_POW.md]: ~~pow(x, y[, z])~~ `pow(x, y[, z])`
~~built in~~ built-in

File: Python-Function_POW.md [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+
+#Argument
```
Comment (theflametrooper) [inline Python-Function_POW.md]: ~~#Argument~~ # Argument

File: Python-Function_POW.md [hunk 3]
```diff
@@ -0,0 +1,21 @@
+# Python 'pow(x,y)'
+
+pow(x, y[, z]) is a built in function in Python 3 to calculate x to the power y and if z is present , returns x to the power y modulo [https://processing.org/reference/modulo.html] z
+
+#Argument 
+The arguments must have numeric types.
+It takes two argument x and y as well as three x,y and z.
```
Comment (theflametrooper) [inline Python-Function_POW.md]: ~~It takes two argument x and y as well as three x,y and z.~~ This function takes two arguments, `x` and `y`, as well as three, `x`,`y` and `z`.

File: Python-Function_POW.md [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+If z is present, x and y must be of integer types, and y must be non-negative.
+
+
+##Return
```
Comment (theflametrooper) [inline Python-Function_POW.md]: ~~##Return~~ ## Return

File: Python-Function_POW.md [hunk 5]
```diff
@@ -0,0 +1,21 @@
...
+If z is present it returns x to the power y modulo z.If only x and y are present it returns x to the power y (same as x**y).
```
Comment (theflametrooper) [inline Python-Function_POW.md]: ~modulo z.If only~~ modulo z. If only

File: Python-Function_POW.md [hunk 6]
```diff
@@ -0,0 +1,21 @@
...
+It is much faster than x**y % z.
+
```
Comment (theflametrooper) [inline Python-Function_POW.md]:
```diff
-
```

File: Python-Function_POW.md [hunk 7]
```diff
@@ -0,0 +1,21 @@
...
+
+##Code Sample
```
Comment (theflametrooper) [inline Python-Function_POW.md]: ~~##Code Sample~~ ## Code Example

File: Python-Function_POW.md [hunk 8]
```diff
@@ -0,0 +1,21 @@
...
+print(pow(2,4))  # prints 16
```
Comment (theflametrooper) [inline Python-Function_POW.md]:
```python
print(pow(2,4))  # prints 16
```

File: Python-Function_POW.md [hunk 9]
```diff
@@ -0,0 +1,21 @@
...
+print(pow(10,-2)) # prints 0.01
```
Comment (theflametrooper) [inline Python-Function_POW.md]: ~~print(pow(10,-2)) # prints 0.01~~

```python
print(pow(10,-2)) # prints 0.01
```

File: Python-Function_POW.md [hunk 10]
```diff
@@ -0,0 +1,21 @@
...
+print(pow(4,3,5)) #prints 4
```
Comment (theflametrooper) [inline Python-Function_POW.md]: ~~print(pow(4,3,5)) #prints 4~~

```python
print(pow(4,3,5)) # prints 4
```

File: Python-Function_POW.md [hunk 11]
```diff
@@ -0,0 +1,21 @@
...
+
+[official docs]https://docs.python.org/3/library/functions.html#pow
```
Comment (theflametrooper) [inline Python-Function_POW.md]: ~~[official docs]https://docs.python.org/3/library/functions.html#pow~~ `[Official Documentation](https://docs.python.org/3/library/functions.html#pow)`

File: Python-Function_POW.md [hunk 12]
```diff
@@ -0,0 +1,21 @@
+# Python 'pow(x,y)'
```
Comment (Rafase282) [inline Python-Function_POW.md]:
```diff
-# Python 'pow(x,y)'
+# Python pow(x,y)
```

Also rename the file:

```diff
-Python-Function_POW
+Python-Function-Pow
```

File: Python-Function-Pow.md [hunk 13]
```diff
@@ -0,0 +1,22 @@
+# Python pow(x,y)
+
+`pow(x, y[, z])` is a built-in function in Python 3 to calculate `x` to the power `y` and if `z` is present , returns `x` to the power `y` [modulo](https://processing.org/reference/modulo.html) `z`
+
+## Arguments
```
Comment (Rafase282) [inline Python-Function-Pow.md]:
```diff
-## Arguments
+## Arguments
+
```

Meaning add a new empty line after each heading and each code block.

File: Python-Function-Pow.md [hunk 14]
```diff
@@ -0,0 +1,22 @@
...
+If `z` is present it returns `x` to the power `y` modulo `z`.If only `x` and `y` are present it returns `x` to the power `y` (same as x**y).
+
+
+
+## Example
+```python
+print(pow(2,4))    # prints 16
+print(pow(10,-2))  # prints 0.01
+print(pow(4,3,5))  # prints 4
+```
```
Comment (Rafase282) [inline Python-Function-Pow.md]: Please add a repl.it snippet with the working code

File: Python-Function-Pow.md [hunk 15]
```diff
@@ -0,0 +1,27 @@
...
+
+
+## Example
+
+```python
+print(pow(2,4))    # prints 16
+print(pow(10,-2))  # prints 0.01
+print(pow(4,3,5))  # prints 4
+```
+:rocket: [REPL It!](https://repl.it/CTGi)
```
Comment (Rafase282) [inline Python-Function-Pow.md]:
```diff
-:rocket: [REPL It!](https://repl.it/CTGi)
+:rocket: [Run Code](https://repl.it/CTGi)
```

**Comment by Rafase282** [comment]
@nj4710 thanks for the update, I made a little adjustment for the repl, we changed them to Run Code on all pages now.

Also, could you link this article to the [Python Functions](https://github.com/FreeCodeCamp/wiki/blob/master/Python-Functions.md) one? They are in alphabetical order there.

**Comment by nj4710** [comment]
Should I add power function in that article?

**Comment by Rafase282** [comment]
Yes please. Just add the link there and change the repl text and I'll merge.

**Comment by nj4710** [comment]
I have changed the repl text and also added link in the article. Please let me know if there are any more changes

**Comment by nj4710** [comment]
That was my first patch. Thank You so much

**Comment by Rafase282** [comment]
Thank you for your contribution.

**Pull request merged**